### ISSUE TITLE
QrPainter emptyColor has no effect

### ISSUE THREAD

**Issue by jjjachyty** [lukef/qr.flutter#60]
```
var qrcode = new QrPainter(
          data: user.uuid,
          version: QrVersions.auto,
          //  embeddedImage: Image(),
          color: Colors.indigo,
          emptyColor: Colors.blue,
          // size: 320.0,
          embeddedImageStyle: QrEmbeddedImageStyle(
            size: Size(60, 60),
          ),
        );
```

the emptyColors not work,and i save image with png,the image background colors was black
by the way:how can i use embeddedImage added to the center of the QR code?

**Comment by jjjachyty** [comment]
i was solve embeddedImage

**Comment by lukef** [comment]
I'll check this out but `emptyColor` is also deprecated in favor of just putting the qr code inside a container that has a background color.

**Comment by lukef** [comment]
This will be fixed in the `3.0.2` release.

**Comment by lukef** [comment]
Next version will be `3.1.0` because I also needed to bump the Flutter version due to incompatibilities (breaking changes in Flutter).
